ofs/apps/cpeng/hps.cpp

```diff
@@ -92,6 +104,10 @@ class cpeng : public opae::afu_test::command
       ->transform(CLI::AsNumberWithUnit(units));
     app->add_option("-c,--chunk", chunk_, "Chunk size. 0 indicates no chunks")
       ->default_str(std::to_string(chunk_));
+    app->add_option("-r,--data-request-limit",
+                    "data request limit is pcie transfer width")
+      ->default_str(std::to_string(data_request_limit_))
+      ->check(CLI::IsMember(limits));
```

Comment:
Isn't this going to restrict the value of --data-request-limit to one of the enumeration values, ie 0b00, 0b01, 0b10, 0b11?

Reply:
yes